scripts/update_imagery.js

```diff
@@ -158,6 +158,10 @@ for (const [sourceID, source] of sources) {
     ];
   }
 
+  if (source.zoomRange) {
```

Comment:
Looking through the [editor-layer-index schema](https://github.com/osmlab/editor-layer-index/blob/gh-pages/schema.json) I don't think we would get source imagery with a `zoomRange` property, so this block would never run?

Reply:
oops, didn't see this, see https://github.com/facebook/Rapid/pull/1496#issuecomment-2263168116 for the reasoning